Source/WTF/wtf/StackTrace.h

```diff
@@ -50,21 +50,22 @@ class PrintStream;
 class StackTrace {
     WTF_MAKE_FAST_ALLOCATED;
 public:
-    WTF_EXPORT_PRIVATE static std::unique_ptr<StackTrace> captureStackTrace(int maxFrames, int framesToSkip = 0);
+    WTF_EXPORT_PRIVATE NEVER_INLINE static std::unique_ptr<StackTrace> captureStackTrace(int maxFrames, int framesToSkip = 0);
```

Comment:
You also need to make `WTFGetBacktrace` NEVER_INLINE.  The "+ 2" that we add to `framesToSkip` is because of `captureStackTrace` and `WTFGetBacktrace`.